sdk/servicebus/azure-servicebus/setup.py

```diff
@@ -20,21 +20,6 @@
 # a-b-c => a.b.c
 namespace_name = PACKAGE_NAME.replace('-', '.')
 
-# azure v0.x is not compatible with this package
-# azure v0.x used to have a __version__ attribute (newer versions don't)
-try:
-    import azure
-    try:
-        ver = azure.__version__
-        raise Exception(
```

Comment:
why remove this?

Reply:
as per this: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/packaging.md#note-on-checking-old-azure-packages